src/components/POLine/LineLinkedInstances/formatters.js

```diff
@@ -0,0 +1,29 @@
+export const formatPublishers = (publishers = []) => {
+  return publishers
+    .filter(Boolean)
+    .map(p => {
+      const dateOfPublication = p.dateOfPublication ? ` (${p.dateOfPublication})` : '';
+
+      return `${p.publisher}${dateOfPublication}`;
+    })
+    .join(', ');
+};
+
+export const formatContributors = (contributors = []) => {
+  return contributors
+    .filter(Boolean)
+    .map(({ name }) => name)
+    .join('; ');
+};
+
+export const formatRelations = (parents = [], children = [], relationTypes = []) => {
+  const relationTypesMap = relationTypes.reduce((acc, rType) => {
+    acc[rType.id] = rType.name;
+
+    return acc;
+  }, {});
```

Comment:
```suggestion
  const relationTypesMap = Object.fromEntries(relationTypes.map(rt => [rt.id, rt.name]);
```

Reply:
I'm not worry about performance, but reduce approach has less complexity + browser support is better (not sure we transpile fromEntries - probably not our case)